engine/common/insight_log/insight_logs_enabled_check.py

```diff
@@ -1,15 +1,20 @@
-from django.apps import apps
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
 
 
 def is_insight_logs_enabled(organization):
     """
     is_insight_logs_enabled checks if inside logs enabled for given organization.
+    Now it checks if oncall is deployed on same cluster that its grafana instance to be able to forward logs.
+    Or if it's Open Source :)
     """
-    DynamicSetting = apps.get_model("base", "DynamicSetting")
-    org_id_to_enable_insight_logs, _ = DynamicSetting.objects.get_or_create(
-        name="org_id_to_enable_insight_logs",
-        defaults={"json_value": []},
+    logger.info(
+        "is_insight_logs_enabled: "
```

Comment:
possible duplication: name of the task will be added to the log by logger

Reply:
This code executes mainly not from tasks